tests/events.test.ts

```diff
@@ -62,45 +57,46 @@ describe("Event related tests", () => {
       id: "",
       key: "issue_comment.created",
       octokit: {
-        issues,
         rest: {
+          issues,
           repos: {
-            getContent() {
-              return {
-                data: `
-                  plugins:
-                    - name: "Run on comment created"
-                      uses:
-                        - id: plugin-A
-                          plugin: https://plugin-a.internal
-                    - name: "Some Action plugin"
-                      uses:
-                        - id: plugin-B
-                          plugin: ubiquibot/plugin-b
-                  `,
-              };
+            getContent(params?: RestEndpointMethodTypes["repos"]["getContent"]["parameters"]) {
+              if (params?.path === ".github/.ubiquibot-config.yml") {
```

Comment:
Perhaps we can import the single constant for this string from the other file above.